### PR TITLE
Entity Token Endpoints

### PR DESCRIPTION
**Pull request by Waidhoferj**
## What's New?
Currently, there isn't a centralized source of information for the Entity tokens used in the QuestionAnswerPairs. For example:
```
Where is [SECRET_HIDEOUT]?
```
While this is totally hilarious as someone who was there for the inside joke, it won't be as intuitive for those who just joined the club.

The API is centered around the endpoint: ``schema/entity_tokens``
#### ``GET`` Pulls down all approved QAPair tokens
Requires no arguments
Returns an array of all tokens:
```
[
{name: "Location", id: "SECRET_HIDEOUT", description: "A place on campus"}
]
```
#### ``POST`` Adds a new key to the ``EntityTokens`` table
Post json body format: ``{name: "Location", id: "SECRET_HIDEOUT", description: "A place on campus"}``

## Type of change (pick-one)
- [ ] Bug fix (_non-breaking change which fixes an issue_)
- [x] New feature (_non-breaking change which adds functionality_)
- [ ] Breaking change (_fix or feature that would cause existing functionality to not work as expected_)
- [ ] This change requires a documentation update

## How Has This Been Tested?
I used Postman to test both endpoints. You can find both requests in the CSAI Postman ``CSAI Web`` folder:
- Get Entity Tokens
- Add Entity Token

### CODE REVIEW

**Comment by Waidhoferj** [comment]
You might also notice that I added the scaffolding for the new formatting schema defined in #174

File: database_wrapper.py [hunk 1]
```diff
@@ -634,6 +636,26 @@ def validate_and_format_entity_data(self, entity_type, data_dict: dict):
...
+        ---------
+        `entity - Entity` an initialized entity object to be added to the database.
+
+        Returns
+        -------
+        `bool` whether entity was successfully added.
+        """
+        # Don't post if the entity doesn't abide by the rules of the Entity superclass
+        if not isinstance(entity, Entity):
+            return False
```
Comment (Jason-Ku) [inline database_wrapper.py:652]: Should this throw an error? Or is the caller expected to handle this

Comment (Waidhoferj) [reply database_wrapper.py:652]: We could do either. I just wanted to make sure that the only objects that get through are a subclass of Entity

Comment (Jason-Ku) [reply database_wrapper.py:652]: Right. If you think we are never going to want to ignore this error, I'd handle the error checking in the callee to save some time/effort in not error checking every time we call it. That's my opinion though - I'll leave it up to your judgement

**Review by snekiam — APPROVED** [review]
This looks great to me! I think this will be super helpful for ongoing support of the API.

**Review by Jason-Ku — APPROVED** [review]

**Pull request merged**